<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="14" passed="8" failed="6" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-20T10:07:20Z" name="Odev" finished-at="2021-12-20T10:18:25Z" duration-ms="664323">
    <groups>
    </groups>
    <test started-at="2021-12-20T10:07:20Z" name="ChromeTest" finished-at="2021-12-20T10:12:55Z" duration-ms="334187">
      <class name="UI.testCases.TestClass">
        <test-method is-config="true" signature="startReport()[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:07:20Z" name="startReport" finished-at="2021-12-20T13:07:21Z" duration-ms="174" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method signature="blankEmail()[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:07:21Z" name="blankEmail" description="Boş Email ile Giriş" finished-at="2021-12-20T13:07:25Z" duration-ms="4083" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankEmail -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:07:25Z" name="getResult" finished-at="2021-12-20T13:07:25Z" duration-ms="280" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=blankEmail status=SUCCESS method=TestClass.blankEmail()[pri:0, instance:UI.testCases.TestClass@1b1473ab] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="blankEmailandPassword()[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:07:25Z" name="blankEmailandPassword" description="Boş Email ve Password ile Giriş" finished-at="2021-12-20T13:07:29Z" duration-ms="3949" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankEmailandPassword -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:07:29Z" name="getResult" finished-at="2021-12-20T13:07:29Z" duration-ms="115" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=blankEmailandPassword status=SUCCESS method=TestClass.blankEmailandPassword()[pri:0, instance:UI.testCases.TestClass@1b1473ab] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="blankPassword()[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:07:29Z" name="blankPassword" description="Boş Password ile Giriş" finished-at="2021-12-20T13:07:32Z" duration-ms="3264" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Hata : Beklenen Veri Hatalı 
Expected: is "Lütfen şifrenizi giriniz."
     got: "Lütfen geçerli bir e-posta adresi giriniz."
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Hata : Beklenen Veri Hatalı 
Expected: is "Lütfen şifrenizi giriniz."
     got: "Lütfen geçerli bir e-posta adresi giriniz."

	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:21)
	at UI.helpers.BrowserUtils.assertControl(BrowserUtils.java:74)
	at UI.testCases.TestClass.blankPassword(TestClass.java:59)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankPassword -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:07:32Z" name="getResult" finished-at="2021-12-20T13:07:33Z" duration-ms="600" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=blankPassword status=FAILURE method=TestClass.blankPassword()[pri:0, instance:UI.testCases.TestClass@1b1473ab] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="butikLinkAddCSV()[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:07:33Z" name="butikLinkAddCSV" description="Butik Link Search" finished-at="2021-12-20T13:09:10Z" duration-ms="96824" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- butikLinkAddCSV -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:09:10Z" name="getResult" finished-at="2021-12-20T13:09:10Z" duration-ms="209" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=butikLinkAddCSV status=SUCCESS method=TestClass.butikLinkAddCSV()[pri:0, instance:UI.testCases.TestClass@1b1473ab] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="loginSucces()[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:09:10Z" name="loginSucces" description="Başarılı Kullanıcı Girişi" finished-at="2021-12-20T13:09:26Z" duration-ms="16136" status="FAIL">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for UI.helpers.BrowserUtils$1@7f065f9e (tried for 10 second(s) with 1000 milliseconds interval)]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for UI.helpers.BrowserUtils$1@7f065f9e (tried for 10 second(s) with 1000 milliseconds interval)
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:303)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:271)
	at UI.helpers.BrowserUtils.fluentWait(BrowserUtils.java:23)
	at UI.testCases.TestClass.loginSucces(TestClass.java:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[contains(text(),'Hesabım')]"}
  (Session info: chrome=96.0.4664.110)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'MAC-IT-13.local', ip: 'fe80:0:0:0:181c:e107:fe30:ce61%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.15.7', java.version: '14.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 96.0.4664.110, chrome: {chromedriverVersion: 96.0.4664.45 (76e4c1bb2ab46..., userDataDir: /var/folders/p5/h99g2kwx3bb...}, goog:chromeOptions: {debuggerAddress: localhost:59518}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: a49f19f1ef813a4cc5bef2b006010c21
*** Element info: {Using=xpath, value=//*[contains(text(),'Hesabım')]}
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:319)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:421)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:311)
	at UI.helpers.BrowserUtils$1.apply(BrowserUtils.java:25)
	at UI.helpers.BrowserUtils$1.apply(BrowserUtils.java:23)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:248)
	... 30 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginSucces -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:09:26Z" name="getResult" finished-at="2021-12-20T13:09:27Z" duration-ms="516" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginSucces status=FAILURE method=TestClass.loginSucces()[pri:0, instance:UI.testCases.TestClass@1b1473ab] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="wrongEmail()[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:09:27Z" name="wrongEmail" description="Hatalı Email ile Giriş" finished-at="2021-12-20T13:11:11Z" duration-ms="104019" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Hata : Beklenen Veri Hatalı 
Expected: is "HATALI CASE"
     got: "E-posta adresiniz ve/veya şifreniz hatalı."
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Hata : Beklenen Veri Hatalı 
Expected: is "HATALI CASE"
     got: "E-posta adresiniz ve/veya şifreniz hatalı."

	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:21)
	at UI.helpers.BrowserUtils.assertControl(BrowserUtils.java:74)
	at UI.testCases.TestClass.wrongEmail(TestClass.java:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- wrongEmail -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:11:11Z" name="getResult" finished-at="2021-12-20T13:11:11Z" duration-ms="511" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=wrongEmail status=FAILURE method=TestClass.wrongEmail()[pri:0, instance:UI.testCases.TestClass@1b1473ab] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="wrongPassword()[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:11:11Z" name="wrongPassword" description="Hatalı Şifre ile Giriş" finished-at="2021-12-20T13:12:54Z" duration-ms="103352" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- wrongPassword -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@1b1473ab]" started-at="2021-12-20T13:12:54Z" name="getResult" finished-at="2021-12-20T13:12:55Z" duration-ms="142" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=wrongPassword status=SUCCESS method=TestClass.wrongPassword()[pri:0, instance:UI.testCases.TestClass@1b1473ab] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- UI.testCases.TestClass -->
    </test> <!-- ChromeTest -->
    <test started-at="2021-12-20T10:12:55Z" name="FirefoxTest" finished-at="2021-12-20T10:18:25Z" duration-ms="330121">
      <class name="UI.testCases.TestClass">
        <test-method is-config="true" signature="startReport()[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:12:55Z" name="startReport" finished-at="2021-12-20T13:12:55Z" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method signature="blankEmail()[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:12:55Z" name="blankEmail" description="Boş Email ile Giriş" finished-at="2021-12-20T13:12:58Z" duration-ms="3458" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankEmail -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:12:58Z" name="getResult" finished-at="2021-12-20T13:12:58Z" duration-ms="105" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=blankEmail status=SUCCESS method=TestClass.blankEmail()[pri:0, instance:UI.testCases.TestClass@4a22f9e2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="blankEmailandPassword()[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:12:58Z" name="blankEmailandPassword" description="Boş Email ve Password ile Giriş" finished-at="2021-12-20T13:13:02Z" duration-ms="3570" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankEmailandPassword -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:13:02Z" name="getResult" finished-at="2021-12-20T13:13:02Z" duration-ms="100" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=blankEmailandPassword status=SUCCESS method=TestClass.blankEmailandPassword()[pri:0, instance:UI.testCases.TestClass@4a22f9e2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="blankPassword()[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:13:02Z" name="blankPassword" description="Boş Password ile Giriş" finished-at="2021-12-20T13:13:06Z" duration-ms="4100" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Hata : Beklenen Veri Hatalı 
Expected: is "Lütfen şifrenizi giriniz."
     got: "Lütfen geçerli bir e-posta adresi giriniz."
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Hata : Beklenen Veri Hatalı 
Expected: is "Lütfen şifrenizi giriniz."
     got: "Lütfen geçerli bir e-posta adresi giriniz."

	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:21)
	at UI.helpers.BrowserUtils.assertControl(BrowserUtils.java:74)
	at UI.testCases.TestClass.blankPassword(TestClass.java:59)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- blankPassword -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:13:06Z" name="getResult" finished-at="2021-12-20T13:13:07Z" duration-ms="617" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=blankPassword status=FAILURE method=TestClass.blankPassword()[pri:0, instance:UI.testCases.TestClass@4a22f9e2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="butikLinkAddCSV()[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:13:07Z" name="butikLinkAddCSV" description="Butik Link Search" finished-at="2021-12-20T13:14:42Z" duration-ms="95609" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- butikLinkAddCSV -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:14:42Z" name="getResult" finished-at="2021-12-20T13:14:42Z" duration-ms="194" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=butikLinkAddCSV status=SUCCESS method=TestClass.butikLinkAddCSV()[pri:0, instance:UI.testCases.TestClass@4a22f9e2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="loginSucces()[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:14:42Z" name="loginSucces" description="Başarılı Kullanıcı Girişi" finished-at="2021-12-20T13:14:56Z" duration-ms="13849" status="FAIL">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Expected condition failed: waiting for UI.helpers.BrowserUtils$1@4403cf9a (tried for 10 second(s) with 1000 milliseconds interval)]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for UI.helpers.BrowserUtils$1@4403cf9a (tried for 10 second(s) with 1000 milliseconds interval)
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:303)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:271)
	at UI.helpers.BrowserUtils.fluentWait(BrowserUtils.java:23)
	at UI.testCases.TestClass.loginSucces(TestClass.java:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//*[contains(text(),'Hesabım')]"}
  (Session info: chrome=96.0.4664.110)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'MAC-IT-13.local', ip: 'fe80:0:0:0:181c:e107:fe30:ce61%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.15.7', java.version: '14.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 96.0.4664.110, chrome: {chromedriverVersion: 96.0.4664.45 (76e4c1bb2ab46..., userDataDir: /var/folders/p5/h99g2kwx3bb...}, goog:chromeOptions: {debuggerAddress: localhost:60111}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: cfb2c22d27a4b3e523dfaf6b1f87243a
*** Element info: {Using=xpath, value=//*[contains(text(),'Hesabım')]}
	at jdk.internal.reflect.GeneratedConstructorAccessor59.newInstance(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:319)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:421)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:311)
	at UI.helpers.BrowserUtils$1.apply(BrowserUtils.java:25)
	at UI.helpers.BrowserUtils$1.apply(BrowserUtils.java:23)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:248)
	... 30 more
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginSucces -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:14:56Z" name="getResult" finished-at="2021-12-20T13:14:57Z" duration-ms="493" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginSucces status=FAILURE method=TestClass.loginSucces()[pri:0, instance:UI.testCases.TestClass@4a22f9e2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="wrongEmail()[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:14:57Z" name="wrongEmail" description="Hatalı Email ile Giriş" finished-at="2021-12-20T13:16:40Z" duration-ms="103723" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Hata : Beklenen Veri Hatalı 
Expected: is "HATALI CASE"
     got: "E-posta adresiniz ve/veya şifreniz hatalı."
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Hata : Beklenen Veri Hatalı 
Expected: is "HATALI CASE"
     got: "E-posta adresiniz ve/veya şifreniz hatalı."

	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:21)
	at UI.helpers.BrowserUtils.assertControl(BrowserUtils.java:74)
	at UI.testCases.TestClass.wrongEmail(TestClass.java:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- wrongEmail -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:16:40Z" name="getResult" finished-at="2021-12-20T13:16:41Z" duration-ms="520" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=wrongEmail status=FAILURE method=TestClass.wrongEmail()[pri:0, instance:UI.testCases.TestClass@4a22f9e2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method signature="wrongPassword()[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:16:41Z" name="wrongPassword" description="Hatalı Şifre ile Giriş" finished-at="2021-12-20T13:18:25Z" duration-ms="103657" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- wrongPassword -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:UI.testCases.TestClass@4a22f9e2]" started-at="2021-12-20T13:18:25Z" name="getResult" finished-at="2021-12-20T13:18:25Z" duration-ms="120" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=wrongPassword status=SUCCESS method=TestClass.wrongPassword()[pri:0, instance:UI.testCases.TestClass@4a22f9e2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- UI.testCases.TestClass -->
    </test> <!-- FirefoxTest -->
  </suite> <!-- Odev -->
</testng-results>
